pkg/controller/vsphere/reconciler.go

```diff
@@ -410,7 +410,9 @@ func clone(s *machineScope) (string, error) {
 
 	vmTemplate, err := s.GetSession().FindVM(*s, "", s.providerSpec.Template)
 	if err != nil {
-		return "", err
+		notFoundMsg := "template not found, specify valid value"
```

Comment:
given that `notFoundMsg` doesn't change, i wonder if it's better to make it a const?

```suggestion
		const notFoundMsg = "template not found, specify valid value"
```